### ISSUE TITLE
Can't find isaacgym module

### ISSUE THREAD

**Issue by karlengblom** [inspirai/TimeChamber#5]
Hello,
When I try to run train.py, the following message appears:

Traceback (most recent call last):
  File "train.py", line 33, in <module>
    import isaacgym
ModuleNotFoundError: No module named 'isaacgym'

But the thing is, IsaacGym is installed and works on my computer, I can run IsaacGymEnvs without problem. Thanks a lot for any help.

**Comment by ZiyiLiubird** [comment]
Hello, 

please make sure you have installed isaacgym in the same conda env with timechamber. 

If the problem still exists, could you give me a screenshot of you conda env list?

**Comment by karlengblom** [comment]
Hello, by "isaacgym" I assume you mean the folder isaacgym/Python/isaacgym, right? If the top folder in the isaacgym tar is copied straight to site-packages, there are error messages about not finding the python files.
Anyway, I did that. Then there were error messages about not finding ninja and scipy, so I installed them, and the error message AttributeError: module 'numpy' has no attribute 'float'. Solved by installing numpy==1.22.4.

After that, it works! It's a very cool repo, thanks for sharing it!
